### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Url Rewrite_Inbound Values: Unable to access the expanded list items from the “Macros” control by pressing enter.

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#160]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
**WCAG ID Violated:** WCAG2.1.1

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #C+E-H; #.NETWebsites; #Chrome; #Edge; WCAG2.1.1

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Url Rewrite and press enter
6.	Tab till Create Rule and press enter
7.	Tab till Macros control and press enter
8.	Validate whether all interactive list items are accessible using keyboard.

**Actual Result:**
Unable to access the expanded list items from the “Macros” control using keyboard.

**Expected Result:**
we should be able to access the expanded list items from the “Macros” control using keyboard
When we press enter at the list items content should get displayed in the “substitution url” control

**User Impact:**
Users with cognitive disabilities finds it difficult in working through the application

**Attaching video for your reference:**
[Bug 160_Keyboard.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331433/Bug.160_Keyboard.pptx)

**Comment by jhkimnew** [comment]
Fixed.

@shhsu, FYI.
Spacebar key can be used to select the current item, which will enable the Insert button. 
Enter key can be used to insert the current item and users don't need to click the Insert button.

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
